.github/workflows/general-ci-tests.yml

```diff
@@ -0,0 +1,110 @@
+name: General MOM_interface CI
+
+# This CI workflow tests against the following questions: 
+# 1. Does standalone mom build and run?
+# 2. Does it pass the tests/check_default_params.py test?
+# 3. Do the scripts in cime_config pass the black formatter? 
+# Please see Issue #138 for more information
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the listed branches
+  push:
+    branches: ["main" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  
+  check_standalone_mom_build_and_run_lightweight_examples:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    steps:
+    
+    # Copied from NCAR/MOM6 - Install Basic Build Packages for MOM6
+      - name: Install Ubuntu Linux packages
+        shell: bash
+        run: |
+          echo "::group::Install linux packages"
+          sudo apt-get update
+          sudo apt-get install netcdf-bin
+          sudo apt-get install libnetcdf-dev
+          sudo apt-get install libnetcdff-dev
+          sudo apt-get install openmpi-bin
+          sudo apt-get install libopenmpi-dev
+          sudo apt-get install linux-tools-common
+          sudo apt-get install -y csh
+          echo "::endgroup::"
+          
+      # Checkout CESM (cesm3_0_alpha03a) and externals
+      - name: Checkout CESM and Externals
+        run: |
+          git clone https://github.com/ESCOMP/CESM.git -b cesm3_0_alpha03a cesm3_0_alpha03a
```

Comment:
I don't think we want to hard-code `cesm3_0_alpha03a` here, though trusting the head of the default branch could cause other headaches. Since we really just need to make sure we get a relatively new FMS tag, though, my vote would be to test on the default branch (drop the `-b cesm3_0_alpha03a` argument to `git clone`); for consistency, you should probably also change the name of the directory it's cloned into -- dropping the second `cesm3_0_alpha03a` argument will just create `CESM/` which is probably fine

Reply:
Hey Mike, 

Sounds great, makes sense, will do.

Thanks,
Manish V.